policy/lib/tekton/trusted.rego

```diff
@@ -101,11 +101,26 @@ _task_expires_on(task) := expires if {
 	expires = time.parse_rfc3339_ns(record.expires_on)
 }
 
+_unexpired_records(records) := all_unexpired if {
+	never_expires := [record |
+		some record in records
+		not record.expires_on
```

Comment:
I think it works the same (unless maybe the value of expires_on is `false`), but consider doing it like this for readability:

```suggestion
		"expires_on" in object.keys(obj)
```